apps/frontend/src/components/modal/stepperModal.tsx

```diff
@@ -51,19 +51,19 @@ function StepperModal({ isOpen, onClose }: Props) {
       return;
     }
     if (!applicationData) return;
-    const balanceResponse = await fetch(
-      `/api/get-balance?stxAddress=${stxAddress}`
-    );
-    const { message: balance } = await balanceResponse.json();
-
-    if (
-      parseFloat(balance) * Math.pow(10, 6) <
-      parseFloat(applicationData.paymentTerms.downpayment) * Math.pow(10, 6)
-    ) {
-      toast.error("Insufficient sUSDT balance");
-      onClose();
-      return;
-    }
+    // const balanceResponse = await fetch(
+    //   `/api/get-balance?stxAddress=${stxAddress}`
+    // );
+    // const { message: balance } = await balanceResponse.json();
+
+    // if (
+    //   parseFloat(balance) * Math.pow(10, 6) <
+    //   parseFloat(applicationData.paymentTerms.downpayment) * Math.pow(10, 6)
+    // ) {
+    //   toast.error("Insufficient sUSDT balance");
+    //   onClose();
+    //   return;
+    // }
```

Comment:
Why is this commented out?

Reply:
@dorucioclea so the idea here was to have a validation check for user having sufficient sUSDT balance to pay downpayment to avoid PO creation failure on chain, but the problem with this is stacks API even though its public API hits cors error sometimes, the solution at that point was to use next JS serverless as a proxy but implementing the dynamic network switch in client side i cant pass the client hook value to next JS server easily so for now i disabled this validation.